src/stores.ts

```diff
@@ -229,7 +229,7 @@ function createWalletStateSliceStore(options: {
   intervalSetting?: number
 }): WalletStateSliceStore {
   const { parameter, initialState, intervalSetting } = options
-  const { subscribe, set } = writable(initialState)
+  const { subscribe, set } = writable<string | number | null | undefined>(initialState)
```

Comment:
I had a typescript error because set function from here wanted to accept only `string | number | null` as an argument if I don't explicitly set undefined